manifests/istio-control/istio-discovery/templates/autoscale.yaml

```diff
@@ -1,8 +1,9 @@
+{{ if or (eq .Values.revision "") (not .Values.clusterResources) }}
```

Comment:
this is per revision? Why is it a cluster resource?

Reply:
You're right, fixed.